doc/source/guidelines/logging.rst

```diff
@@ -237,13 +235,13 @@ logger.
 
 Instance logger
 ~~~~~~~~~~~~~~~
-Every time that the class ``_MapdlCore`` is instantiated, a logger is
-created.  This logger is recommended when using the ``pool`` library
-or when using multiple instances of ``Mapdl``.  The main feature of
-this logger is that it tracks each instance and it includes its name
-when logging.  The name of the instances are unique.  For example in
-case of using the ``gRPC`` ``Mapdl`` version, its name includes the IP
-and port of the correspondent instance, making unique its logger.
+Every time that the class ``_MapdlCore`` is instantiated, a logger is created.
+This logger is recommended when using the ``pool`` library or when using
+multiple instances of ``Mapdl``.  The main feature of this logger is that it
+tracks each instance and it includes its name when logging.  The name of the
+instances are unique.  For example in case of using the ``gRPC`` ``Mapdl``
+version, its name includes the IP and port of the correspondent instance, making
+unique its logger.
```

Comment:
@akaszynski I am not sure why we have 2 white blanks after sentence's dot.
```suggestion
multiple instances of ``Mapdl``. The main feature of this logger is that it
tracks each instance and it includes its name when logging. The name of the
instances are unique. For example in case of using the ``gRPC`` ``Mapdl``
version, its name includes the IP and port of the correspondent instance, making
unique its logger.
```

Reply:
> I am not sure why we have 2 white blanks after sentence's dot. I think that we can get rid of those ones. Is it usefull for Emacs?

That's just my 9th grade english showing through. Standard these days is single space. Back in the typewriter days, the standard was two spaces: https://en.wikipedia.org/wiki/Sentence_spacing. Emacs is close to using a typewriter, but even there I can learn how to write one space after a period.